templates/init/dataconnect/schema.gql

```diff
@@ -1,28 +1,46 @@
-# # Example schema for simple email app
-# type User @table(key: "uid") {
-# 	uid: String!
-# 	name: String!
-# 	address: String!
+# # Example schema for simple movie review app
+
+# # Users
+# # Suppose a user can leave reviews for movies
+# # user  -> reviews is a one to many relationship,
+# # movie -> reviews is a one to many relationship
+# # movie <-> user is a many to many relationship
+# type User @table {
+#   id: String! @col(name: "user_auth")
+#   username: String! @col(name: "username", dataType: "varchar(50)")
+#   # The following are generated by the user: User! field in the Review table
+#   # reviews_on_user 
+#   # movies_via_Review
 # }
 
-# type Email @table {
-# 	subject: String!
-# 	sent: Date!
-# 	text: String!
-# 	from: User!
+# # Movies
+# type Movie @table {
+#   # The below parameter values are generated by default with @table, and can be edited manually.
+#   # implicitly calls @col to generates a column name. ex: @col(name: "movie_id")
+#   # for UUID fields, @default(expr: "uuidV4()") is also implicitly called
```

Comment:
I don't think this is true. If the entire `id` field is omitted then `id: UUID! @default(expr: "uuidV4()")` is added, but I don't think we add defaults to explicit UUID-typed fields. Otherwise, there's no way to declare a UUID field _without_ a default, even if the developer wants to.

Feel free to file a bug if you feel like we should either change the behavior or the documentation, or both.

Reply:
That makes sense! Updated!

Reply:
This line doesn't seem to be updated on my side. Is a git commit/push needed?

Reply:
Hmmm, just pushed again, is it updated now?

Reply:
Yes, thanks!